combat-trainer.lic

```diff
@@ -2736,6 +2758,89 @@ class TrainerProcess
          'dumping some dirt', 'What were you referring')
   end
 
+  def pickbox(game_state, number='first')
```

Comment:
I'd feel a lot better about making this an integer and passing along an int everywhere, then using it to lookup the ordinal string right before it's used (e.g. lines 2791/2793).

https://github.com/rpherbig/dr-scripts/blob/5b773a5e09b0a066bde95c5bb6ed69966010d5a0/common.lic#L43 should do it

What do you think?

Reply:
I can try it. I think I did try once but results were not looking good.